tests/test_conjugates.py

```diff
@@ -0,0 +1,40 @@
+import aesara
+import aesara.tensor as at
+import pytest
+from aesara.graph.unify import eval_if_etuple
+from aesara.tensor.random import RandomStream
+from kanren import run
+from kanren.graph import walko
+from unification import var
+
+from aemcmc.conjugates import conjugate
+
+
+def test_beta_binomial_conjugate():
+    """Test that we can produce the closed-form posterior for
+    the binomial observation model with a beta prior.
+
+    """
```

Comment:
Does it work the other way: i.e. can it expand a beta into a binomial and a beta?

Reply:
Great point, I'm still used to thinking one way. I'll expand the test.

Reply:
Don't worry, it doesn't need to right now, but it's good to understand why.